src/App.vue

```diff
@@ -227,6 +227,16 @@ export default {
 		},
 	},
 
+	beforeCreate() {
+		const authorizedUser = getCurrentUser().uid
+		const lastLoggedInUser = BrowserStorage.getItem('last_logged_in_user')
+
+		if (authorizedUser !== lastLoggedInUser) {
+			BrowserStorage.clear()
+		}
+		BrowserStorage.setItem('last_logged_in_user', authorizedUser)
+	},
```

Comment:
For guest `getCurrentUser().uid === null`.

On the first open `authorizedUser === lastLoggedInUser === null` and it works fine. But it saves a string `'null'` as `last_logged_in_user` value.

After a reload `authorizedUser !== lastLoggedInUser` because `null !== 'null'` and everything is cleared. While it's good to clear the cache for guests, maybe we shouldn't clear all the settings. Or at least do it implicitly.

Reply:
> For guest `getCurrentUser().uid === null`.

Actually, `getCurrentUser() === null` and `getCurrentUser().id` throws an error :see_no_evil: 
Thanks for noticing

> On the first open `authorizedUser === lastLoggedInUser === null` and it works fine. But it saves a string `'null'` as `last_logged_in_user` value.

Solvable by pushing a storage value through JSON.parse()

Reply:
> Since we remember camera setting etc as well, could we just wipe the conversation list thing for now?

If we could treat a guest (non-logged) as user with uid === null, we could keep the cache. Guests don't fetch conversations, so it'll never appear for them. And if someone joins as a guest -> `'someone_uid' !== null` -> wipe

Reply:
Just thinking about the session issues we have atm on our instance.
If I happen to rejoin to a public conversation after hybernate, it would wipe all my camera/mic preferences